specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManagerEffectiveConfiguration.json

```diff
@@ -89,7 +89,7 @@
   "definitions": {
     "EffectiveConfiguration": {
```

Comment:
>EffectiveConfiguration" [](http://example.com/codeflow?start=5&length=23)

this should be modeled as a resource with id, name, type properties at least since it is returned via a collection get

Reply:
We can't model it as a resource since it is dynamic. Everytime, we need to generate a new live EffectiveConfiguration when customers call